css/base.css

```diff
@@ -0,0 +1,16 @@
+html {
+  /* outline: 1px solid black; */
```

Comment:
you can remove commented code

Reply:
that's for development use only. 
will be removed in the end.